src/commands/findMembersAt.ts

```diff
@@ -40,17 +40,21 @@ export const cmdFindMembersAt: Middleware<BotContext> = async (
     })
   );
 
-  const message =
-    members.length === 0
-      ? i18n.t('location.noMembersAtLocation', {
-          mention: ctx.safeUser.mention,
-          country: getCountryNameForCountryCode(countryCode),
-        })
-      : i18n.t('location.membersAtLocation', {
-          mention: ctx.safeUser.mention,
-          members: members.join(', '),
-          country: getCountryNameForCountryCode(countryCode),
-        });
-
-  return ctx.replyWithMarkdown(message);
+  const hasNoMembers = members.length === 0;
+
+  const message = hasNoMembers
+    ? i18n.t('location.noMembersAtLocation', {
+        mention: ctx.safeUser.mention,
+        country: getCountryNameForCountryCode(countryCode),
+      })
+    : i18n.t('location.membersAtLocation', {
+        mention: ctx.safeUser.mention,
+        members: members.join(', '),
+        country: getCountryNameForCountryCode(countryCode),
+      });
+
+  return ctx.replyWithAutoDestructiveMessage(message, {
+    deleteReplyMessage: hasNoMembers,
+    deleteCommandMessage: hasNoMembers,
+  });
```

Comment:
here is an example of using the auto destructive message only if a specific condition is met

Reply:
Nice, and now I notice that I was forgetting to delete the command message too